<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>markers</class>
 <widget class="QDialog" name="markers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_StudentSelection">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>311</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rubric</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GradeEssayActivity">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Grade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Marker View</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Rubric">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenInstructorSolution">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Instructor Solution</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Grade</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
